spec/controllers/course/registrations_controller_spec.rb

```diff
@@ -0,0 +1,218 @@
+require 'rails_helper'
+
+RSpec.describe Course::RegistrationsController, :type => :controller do
+  before do
+    ActsAsTenant.current_tenant = Instance.default
+  end
+
+  let!(:user) { create(:user) }
+  let!(:course) { create(:course, status: :opened) }
+  let!(:student_request) { create(:student_registration, course: course, user: create(:user)) }
+
+  let!(:student_request2) { create(:student_registration, course: course, user: create(:user)) }
+  let!(:staff_request) { create(:teaching_assistant_registration, course: course,
+                                user: create(:user)) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 14, 54 should be on its own line.